### PR TITLE
Update labels on search and advanced search options

### LINKED ISSUE #68: Updates to Search page
Screen Shot 2022-06-23 at 12 16 26 PM

Here are my suggestions:

Change "Search the collections by URL:" to "Locate Archived Sites by URL" to better reflect what user can do.
Change "Search" to "Go" 
Remove "Match Type" - the search result is difficult to interpret
Change "Date/Time Range" to "Date Range (YYYY/MM/DD)
Remove  whole section on "Filtering"

### PR DESCRIPTION
**Pull request by aaron-collier**
Closes #68 

Screen Shot 2022-06-27 at 3 29 09 PM

### CODE REVIEW

**Comment by edsu** [comment]
When I used the docker environment to search for the URL *https://apod.nasa.gov/* it didn't seem to work and I got this in my dev console:

```
jQuery.Deferred exception: document.getElementById(...) is null @http://localhost:8080/static/search.js:174:12
l@http://localhost:8080/static/js/jquery-latest.min.js:2:29375
Deferred/then/a/</c<@http://localhost:8080/static/js/jquery-latest.min.js:2:29677
setTimeout handler*Deferred/then/a/<@http://localhost:8080/static/js/jquery-latest.min.js:2:29886
u@http://localhost:8080/static/js/jquery-latest.min.js:2:27457
fireWith@http://localhost:8080/static/js/jquery-latest.min.js:2:28202
fire@http://localhost:8080/static/js/jquery-latest.min.js:2:28238
u@http://localhost:8080/static/js/jquery-latest.min.js:2:27457
fireWith@http://localhost:8080/static/js/jquery-latest.min.js:2:28202
ready@http://localhost:8080/static/js/jquery-latest.min.js:2:31285
_@http://localhost:8080/static/js/jquery-latest.min.js:2:31419
EventListener.handleEvent*@http://localhost:8080/static/js/jquery-latest.min.js:2:31533
@http://localhost:8080/static/js/jquery-latest.min.js:2:220
@http://localhost:8080/static/js/jquery-latest.min.js:2:225
```

Do you see the same?

File: pywb/templates/search.html [hunk 1]
```diff
@@ -23,164 +23,107 @@ <h4 class="display-4">
...
-                    <option value="domain">{% trans %}Domain{% endtrans %}</option>
-                </select>
-            </div>
             <p style="cursor: help;">
-               <span data-toggle="tooltip" data-placement="right"
-                     title="Restricts the results to the given date/time range (inclusive)">
-                   {{ _('Date/Time Range') }}
+                <span data-toggle="tooltip" data-placement="right"
+                    title="Restricts the results to the given date/time range (inclusive)">
+                    {{ _('Date Range (YYYY/MM/DD)') }}
```
Comment (edsu) [inline pywb/templates/search.html:62]: I know that @peterchanws wrote that the format is YYYY/MM/DD in the ticket, but it is strangely YYYYMMDD

**Review by edsu — CHANGES_REQUESTED** [review]

File: pywb/templates/search.html (hunk 1 above)
Comment (aaron-collier) [reply pywb/templates/search.html:62]: Thanks, I'll update. As for the other comment, I didn't test searching just that the page rendered the changes as all this PR changes is the labels... maybe the ID changed because of that (hence the `getElementById` error) /shrug.

Comment (aaron-collier) [reply pywb/templates/search.html:62]: @edsu it appears to be caused (at least) by removing the `Match Type` section in the advanced search options. So it seems we can't just remove them, I'll work on simply making them hidden.

**Review by edsu — APPROVED** [review]

File: pywb/templates/search.html [hunk 2]
```diff
@@ -67,120 +68,126 @@ <h4 class="display-4">
...
-                                <option value="url">{% trans %}URL{% endtrans %}</option>
-                            </select>
+                        <label class="sr-only" for="dt-to" aria-label="Date/Time Range To">{% trans %}To:{% endtrans
+                            %}</label>
+                        <div class="input-group">
+                            <div class="input-group-prepend">
+                                <div class="input-group-text">{% trans %}To:{% endtrans %}</div>
+                            </div>
+                            <input id="dt-to" type="number" name="date-range-to" class="form-control"
+                                pattern="^\d{4,14}$">
```
Comment (mjgiarlo) [inline pywb/templates/search.html:99]: There's a bunch of line break changes in this commit that don't seem related but they also look harmless. I assume this is just your editor keeping lines from getting too long? It's NBD.
